mvnxparser/mvnxStreamReader/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+# Copyright: (C) 2017 iCub Facility
+# Author: Diego Ferigo
+# CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+
+# Load dependencies
+find_package(Qt5Xml REQUIRED)
+
+# Build the libraries
```

Comment:
The library type is managed by `BUILD_SHARED_LIBS` variable, that defaults to ON, hence SHARED. 
To avoid problems in compiling the library as shared in Windows, I suggest moving the calls setting `CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS` (https://github.com/robotology-playground/xsens-mvn/search?utf8=%E2%9C%93&q=CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS&type=) to the root CMakeLists.txt to make sure it is always visible.